metagraph/src/graph/alignment/aligner_extender_methods.hpp

```diff
@@ -96,22 +126,51 @@ class DefaultColumnExtender : public SeedFilteringExtender {
                               char /* last char of node */,
                               ssize_t /* offset (distance from start of the first node) */,
                               ssize_t /* absolute index of maximal value*/,
-                              ssize_t /* trim (starting absolute index of array) */>;
+                              ssize_t /* trim (starting absolute index of array) */,
+                              size_t /* xdrop_cutoffs_ index */,
+                              size_t /* last fork index in table */,
+                              score_t /* traversal penalty */>;
```

Comment:
Anything untrivial like this should be a struct instead of an std::tuple. Especially since you're writing a constructor separately `alloc_column(size_t size, RestArgs... args)`

Reply:
up

Reply:
done